x/archivedb/db.go

```diff
@@ -84,6 +87,11 @@ func NewArchiveDB(
 	}, nil
 }
 
+// Returns the value of a metadata
+func (db *archiveDB) getMetadata(key []byte) ([]byte, error) {
+	return db.rawDB.Get(newMetaKey(key).Bytes())
+}
```

Comment:
Is this planned to be used anywhere? currently we just directly access the DB rather than calling this function.

Reply:
You are right. I'll make it happen